project.py

```diff
@@ -116,6 +116,16 @@ def _append_item_to_project(self, item):
             body = body + '\n<li><b>status</b>: ' + item.status
         except AttributeError:
             pass
+        try:
+            environment_txt = '<b>environment</b>: <code>' + item.environment + '</code>'
+            lines = str(item.environment).splitlines()
+            # Remove empty lines
+            lines = [line for line in lines if line.replace('<br/>', '').strip() != '']
+            if len(lines) > 1:
+                environment_txt = '<details><summary>environment</summary>\n\n```\n' + '\n'.join(lines) + '\n```\n</details>'
+            body = body + '\n<li>' + environment_txt
```

Comment:
Any reason to not put this at the top?

Just follow the standard issue template,

e.g.:
https://github.com/jenkinsci/pipeline-graph-view-plugin/issues/950

https://github.com/jenkinsci/.github/blob/master/.github/ISSUE_TEMPLATE/1-report-bug.yml

Reply:
- Opened https://github.com/lemeurherve/jira-issues-importer/pull/20